benchmark/Graphs.jl

```diff
@@ -281,10 +281,6 @@ lgbench["sum-weights"] = @benchmarkable begin
   total
 end
 
-clvecbench["increment-weights"] = @benchmarkable begin
```

Comment:
We no longer expose g[:weight] as a vector anymore, so this doesn't work. At some point, we may want to expose something like this, but we have to think carefully about exactly what this would mean.

Reply:
To summarize our conversation today: this changes breaks several [downstream packages](https://github.com/AlgebraicJulia/Catlab.jl/actions/runs/3588001436), so we should return an `ACSetColumnView <: AbstractVector` that knows how to `setindex!`.